dashboard/src/components/Badge.tsx

```diff
@@ -39,9 +49,25 @@ export default function Badge(props: BadgeProps): JSX.Element {
   // the default type is "default".
   // the resulting span element is stored in badge_elem.
   const badgeElem = (
-    <span className={`${badgeBase} ${colorVariants[props.type]}`}>
+    <span
+      className={`${badgeBase} ${colorVariants[props.type]} ${
+        props.additionalClassNames ?? ""
+      }`}
+    >
       {props.children}
     </span>
   );
   return badgeElem;
 }
+
+export const getBadgeType = (status: string) => {
+  if (status === "Unknown") {
+    return BadgeCodes.INFO;
+  } else if (status === "Healthy") {
+    return BadgeCodes.SUCCESS;
+  } else if (status === "Progressing") {
+    return BadgeCodes.WARNING;
+  } else {
+    return BadgeCodes.ERROR;
+  }
+};
```

Comment:
why not put the values in the enum keys?

Info = "Unknown",
Success = "Healty"
etc.

then you can just use it instead of this function